### PR TITLE
Add constant visibility in all classes of src

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add constant visibility in all classes of src. Constants with no visibility are public by default. This PR makes this more explicit and [this is a rule we want to enforce](https://github.com/PrestaShop/PrestaShop/issues/22728#issuecomment-760086543) using static analysis, so this PR fixes the code before we enable the rule.
| Type?             | refacto
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     |
| How to test?      | CI is green
| Possible impacts? | 

I submitted this PR by using a new [PHPStan custom rule for PrestaShop](https://github.com/PrestaShop/phpstan-prestashop/pull/5/)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23200)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig [hunk 1]
```diff
@@ -24,7 +24,7 @@
  *#}
 {%
   set moduleImage = module.attributes.img is null ?
-    constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
+    public constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]:
```suggestion
    constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
```

**Review by Progi1984 — COMMENTED** [review]
A small and unique modification.

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig (hunk 1 above)
Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig]: Haha, I tried to read it all to find this kind of mistakes but I missed it!

Bravo 😉  very nice catch

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Pull request merged**